cmake/Modules/Findswig.cmake

```diff
@@ -9,15 +9,12 @@ find_package_handle_standard_args(SWIG DEFAULT_MSG
 if(NOT SWIG_EXECUTABLE)
   find_package(Git REQUIRED)
 
-  set(URL https://github.com/swig/swig)
-  set(VERSION fbeb566014a1d320df972aef965daf042db7db36) # Tag origin/rel-3.0.12
-  set_target_description(swig "Simplified Wrapper and Interface Generator (SWIG)" ${URL} ${VERSION})
+  set(URL http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz)
+  set_target_description(swig "Simplified Wrapper and Interface Generator (SWIG)" ${URL} 3.0.12)
 
   ExternalProject_Add(swig_swig
-      GIT_REPOSITORY ${URL}
-      GIT_TAG        ${VERSION}
-      GIT_PROGRESS   ON
-      PATCH_COMMAND ${GIT_EXECUTABLE} apply ${PROJECT_SOURCE_DIR}/patch/add-nodejs8-support-to-swig.patch || true
+      URL ${URL}
+      PATCH_COMMAND patch -p1 < ${PROJECT_SOURCE_DIR}/../patch/add-nodejs8-support-to-swig.patch || true
```

Comment:
Is `patch` command presented on any MacOS and Linux?

Reply:
What if **Findswig.cmake** module is called from topper level directories and `PROJECT_SOURCE_DIR` points to another directory, not shared_model?

Reply:
That's true at least for Ubuntu 16.04. I have no idea how to check this for macOS.
Findswig will be never called from another project since swig is only needed inside shared model.

Reply:
> That's true at least for Ubuntu 16.04. I have no idea how to check this for macOS.
I checked and it seems as presented, at least on the system with standard build tools required by Iroha SDK. So I think it's OK.